### PR TITLE
cmd: fix comments warning in cmd

### PR DESCRIPTION
**Pull request by knarfeh**
BTW，there are no warnings in meta, domain.
screen shot 2017-05-01 at 11 56 10

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: cmd/benchdb/main.go [hunk 1]
```diff
@@ -275,7 +275,7 @@ func (ut *benchDB) selectRows(spec string) {
 	})
 }
 
-// manually trigger GC and send to done channel after finished.
+// manualGC manually trigger GC and send to done channel after finished.
```
Comment (zimulala) [inline cmd/benchdb/main.go]: s/trigger/triggers
s/send/sends

File: cmd/benchkv/main.go [hunk 2]
```diff
@@ -80,7 +80,7 @@ func Init() {
 	go http.ListenAndServe(":9191", nil)
 }
 
-// without conflict
+// batchRW make sure conflict free
```
Comment (zimulala) [inline cmd/benchkv/main.go]: s/make/makes
Please add `.` at the end of a line.

Comment (knarfeh) [inline cmd/benchkv/main.go]: Updated.

File: cmd/benchdb/main.go (hunk 1 above)
Comment (knarfeh) [inline cmd/benchdb/main.go]: Updated.

File: cmd/benchraw/main.go [hunk 3]
```diff
@@ -34,7 +34,7 @@ var (
 	valueSize = flag.Int("V", 5, "value size in byte")
 )
 
-// blind put bench
+// batchRawPut blind put bench
```
Comment (zimulala) [inline cmd/benchraw/main.go]: s/blind/blinds
Please add `.`

Comment (knarfeh) [inline cmd/benchraw/main.go]: Ha, updated.

**Comment by zimulala** [comment]
LGTM

**Pull request merged**